blocks/editable/index.js

```diff
@@ -194,10 +194,41 @@ export default class Editable extends wp.element.Component {
 	}
 
 	onKeyDown( event ) {
+		const { keyCode } = event;
+		const moveUp = ( keyCode === UP || keyCode === LEFT ) && this.isStartOfEditor();
+		const moveDown = ( keyCode === DOWN || keyCode === RIGHT ) && this.isEndOfEditor();
+		const selectors = [
+			'*[contenteditable="true"]',
+			'*[tabindex]',
```

Comment:
Yeah, this won't accommodate all possibilities. Another being `contenteditable` without an explicitly `true` value (`contenteditable`, `contenteditable=""`). There's also `select`, `button`, `object`, `area`, `a`, though dunno if we want to test all of them in this behavior.

I'm forgetting the resource I used in creating it, but this implementation is close to spec for "focusable":

https://codepen.io/aduth/pen/wdRzxM

Maybe also worth moving out as a separate utility.